cmd/ingress-operator/start.go

```diff
@@ -71,7 +71,7 @@ func NewStartCommand() *cobra.Command {
 }
 
 func start(opts *StartOptions) error {
-	metrics.DefaultBindAddress = ":60000"
+	metrics.DefaultBindAddress = ":60001"
```

Comment:
What's this? Do we have some other unreported bug on master?

Reply:
@ironcladlou I'm running the operator locally and port 6000 is already in use. I'll remove.